raster3d/r3.in.v5d/v5d.c

```diff
@@ -1967,7 +1968,9 @@ int v5dReadCompressedGrid(v5dstruct *v, int time, int var, float *ga, float *gb,
 
     /* move to position in file */
     pos = grid_position(v, time, var);
-    lseek(v->FileDesc, pos, SEEK_SET);
+    if (lseek(v->FileDesc, pos, SEEK_SET) == -1) {
+        G_fatal_error("Seek error: %s", strerror(errno));
```

Comment:
In https://github.com/OSGeo/grass/pull/4151, the message to return when using lseek was suggested to be standardized with other messages. I think it should be the case here too.

Reply:
@ShubhamDesai Please, address this just like the other PR.